### ISSUE TITLE
Support most common string commands

### ISSUE THREAD

**Issue by 72squared** [scoquelin/arugula#12]
Here is a list of all the commands in the lettuce `RedisStringAsyncCommands` class.

- append
- bitcount
- bitcount
- bitfield
- bitpos
- bitpos
- bitopNot
- bitopOr
- bitopXor
- decr
- decrby
- get
- getbit
- getdel
- getex
- getrange
- getset
- incr
- incrby
- incrbyfloat
- mget
- mset
- msetnx
- set
- setGet
- setbit
- setex
- psetex
- setnx
- setrange
- stralgoLcs
- strlen

start by adding a comment for the missing commands, and then start adding support for the most commonly used ones.
Here is a suggested list to start with.

- decr
- decrby
- getdel
- getex
- getset
- incr
- incrby
- incrbyfloat
- psetex
- strlen

could also add support for the mget and mset type commands since they are commonly used but they don't work well with cluster unless you take care to map they keys all to the same shard.

**Comment by 72squared** [comment]
probably need to split up the tests by type ... will make it easier to manage as the commands grow

**Comment by 72squared** [comment]
Working on these commands next.

- append
- getex
- getrange
- mget
- mset
- msetnx
- set
- setGet
- setex
- psetex
- setrange
- stralgoLcs
- strlen

**Comment by 72squared** [comment]
The only command left to implement is `stralgoLcs` ... I actually implemented it and found that when I ran the integration tests, the underlying redis 7.0 instance no longer supports it.

https://github.com/redis/redis/issues/9744

The `STRALGO LCS` command was removed and alternative functionality was implemented in the `LCS` command. When Lettuce goes back and fixes this, we can come back and clean up this one command. Otherwise, I think strings are good to go.
